src/pytest/test_replicate.py

```diff
@@ -39,15 +39,14 @@ def checkSlaveSynced(env, slaveConn, command, expected_result, time_out=5):
 
 def testDelReplicate():
   env = Env(useSlaves=True, forceTcp=True)
+  # temp skip for out-of-index
   env.skipOnCluster()
 
   master = env.getConnection()
   slave = env.getSlaveConnection()
   env.assertTrue(master.execute_command("ping"))
   env.assertTrue(slave.execute_command("ping"))
-  env.assertOk(master.execute_command('ft.create', 'idx', 'schema', 'f', 'text'))
-
-  checkSlaveSynced(env, slave, ('exists', 'idx:idx'), 1, time_out=20)
```

Comment:
Why is this not needed?

Reply:
Was removed with initial out-of-keyspace commit.
Need a closer look. sleep does not fix